core/src/main/scala/org/http4s/Message.scala

```diff
@@ -522,15 +522,30 @@ object Request {
   *                   parameters which may be used by the http4s backend for
   *                   additional processing such as java.io.File object
   */
-final case class Response[F[_]](
-    status: Status = Status.Ok,
-    httpVersion: HttpVersion = HttpVersion.`HTTP/1.1`,
-    headers: Headers = Headers.empty,
-    body: EntityBody[F] = EmptyBody,
-    attributes: Vault = Vault.empty)
+final class Response[F[_]](
```

Comment:
Is there any reason for the constructor to remain public?  We could also get rid of the redundant defaults then.

Reply:
I left it like that because I followed `Request`. Should I change them both?

Reply:
I think we should.